partiql-lang/src/main/kotlin/org/partiql/lang/planner/GlobalVariableResolver.kt

```diff
@@ -56,6 +72,18 @@ fun interface GlobalVariableResolver {
      */
     fun resolveGlobal(bindingName: BindingName): GlobalResolutionResult
```

Comment:
I think you can drop this--it is redundant when you have `resolveGlobal(BindingId)`.  Or, more accurately, drop this function and move its documentation (with modification) to the other overload.  Then, remove the default implementation of `resolveGlobal(BindingId)` so we're forced to implement it.

Reply:
For backwards compatibility purposes, when adding new APIs, we've been adding default implementations that "may" utilize existing implemented methods, so that users who don't want to update their code don't have to. That being said, you're the only customer here, so it's up to you.